internal/query/authn_key.go

```diff
@@ -180,7 +180,8 @@ func (q *Queries) searchAuthNKeys(ctx context.Context, queries *AuthNKeySearchQu
 		AuthNKeyColumnEnabled.identifier():    true,
 		AuthNKeyColumnInstanceID.identifier(): authz.GetInstance(ctx).InstanceID(),
 	}
-	stmt, args, err := query.Where(eq).ToSql()
+
+	stmt, args, err := query.Where(eq).OrderBy(AuthNKeyColumnCreationDate.identifier()).ToSql()
```

Comment:
Please remove the hard coded ordering. If we need a default or some ordering for the management part, we need to add it there specifically. There's the possibility to pass the ordering at least in the v2 apis, which would be ignored now.